### ISSUE TITLE
invalid escape character in windows

### ISSUE THREAD

**Issue by ittayd** [andypetrella/spark-notebook#642]
using 0.63, graphx example:

:local-repo /tmp/spark-notebook/
<console>:3: error: invalid escape character
 "Repo changed to C:\tmp\spark-notebook!"
                         ^

**Comment by andypetrella** [comment]
aouch usual suspect under windows... anyway this step isn't necessary anymore, you can skip the :local-repo thing, and probably also the :deps one
need to change that though :-(

**Comment by ittayd** [comment]
There are other examples that work well under linux and not under windows.
For example, use of syslog to demonstrate Spark. Would be nice to have all
examples working by downloading any required resource or bundling it inside
the package and not using linux system commands

**Comment by andypetrella** [comment]
yes there is an issue for that on which I'm very open to help if necessary (don't have windows... so I cannot validate my changes ^^) (see #458)
